Lib/threading.py

```diff
@@ -3,6 +3,7 @@
 import os as _os
 import sys as _sys
 import _thread
+import warnings
```

Comment:
I would prefer to avoid always importing warnings, since up to date code is not supposed to emit a warning. I suggest to move the import inside the function calls (in each isAlive() function).